src/database/schema/ChannelListSchema.ts

```diff
@@ -243,6 +249,7 @@ class ChannelList implements BaseDbSchema {
       LEFT JOIN ${UserSchema.getTableName()} B
       ON A.last_updated_by = B.user_id AND A.id = B.channel_id
       WHERE expired_at IS NULL OR datetime(expired_at) >= datetime('now') AND A.description != ''
+      AND A.channel_type NOT IN ('ANON_PM', 'ANON_POST_NOTIFICATION', 'ANON_GROUP')
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The SQL query in this function has been modified to exclude certain channel types. However, the logic of the WHERE clause might not work as expected due to operator precedence. The `AND` operator has higher precedence than `OR`, so the current query might not filter out expired channels correctly. Consider adding parentheses to ensure the correct execution order.
```diff
- WHERE expired_at IS NULL OR datetime(expired_at) >= datetime('now') AND A.description != ''
+ WHERE (expired_at IS NULL OR datetime(expired_at) >= datetime('now')) AND A.description != ''
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->